Shreya/HTML Basics Remake/form.html

```diff
@@ -8,22 +9,22 @@
         <h1>Register</h1>
             <p>
                 <label for="first">First Name</label>
-                <input type="text" name="first" placeholder="First Name" required>
+                <input type="text" id="first" placeholder="First Name" required>
                 <label for="last">Last Name</label>
-                <input type="text" name="last" placeholder="Last Name" required>
+                <input type="text" id="last" placeholder="Last Name" required>
```

Comment:
Why change from `name` to `id`? Revert it back to `name`

Refer this: [What is the difference between id and name attributes in form elements? - Treehouse](https://teamtreehouse.com/community/what-is-the-difference-between-id-and-name-attributes-in-form-elements)

Reply:
okay